### ISSUE TITLE
Reproducing paper results

### ISSUE THREAD

**Issue by ethanluoyc** [princeton-nlp/WebShop#26]
## I'm submitting a ...

- [ ] bug report
- [x] feature request

## What is the current behavior?

Hi I would like to know if we should expect the pretrained models provided in this repo to reproduce the results from the paper?

### Steps to Reproduce

I can the `test.py` script with the pretrained models provided (and default args), but I wasn't able to reproduce the results from the paper.

## What is the expected behavior?

Can the author clarify if the pretrained models should be able to reproduce results from the paper.

## Motivation for Change
NA

## Please tell us about your environment:
NA

**Comment by haoyangcai0711** [comment]
Same here. I tried to run the IL checkpoints (search model: choice_il_epoch9.pth and choice model: search_il_checkpoint_800/checkpoint-800/) provided by the author using test.py, with all hyper-parameters set to default. However I only got the following result. 

------
avg test score (model, rule): 28.162666666666667 26.697666666666656
avg test success rate % (model, rule): 7.000000000000001 3.8

Did I make any mistakes when reproducing the baseline?

**Comment by ysymyth** [comment]
hi, is it possible to upload some trajectories for debugging? make sure you have followed everything in https://github.com/princeton-nlp/WebShop#-setup

**Comment by ysymyth** [comment]
close it for now but feel free to open new ones

**Comment by ai-nikolai** [comment]
@haoyangcai0711 @ethanluoyc did you guys manage to reproduce results?

@ysymyth on another note - do you know what the current results using ReAct are on Webshop using the latest OpenAI models?

Could you run these somehwere and post them? (We tried to get anything close to decent performance using the official ReAct repo and other repos and get terrible scores these days).

**Comment by ZHUWEI-hub** [comment]
Hello, I am also trying to reproduce this series of papers. In the webshop environment, my fixed_0-fixed_100 is different from the instructions in React and Reflexion papers. I noticed that their instructions in fixed_0-fixed_100 are the same.
I have used my webshop/ setup.sh -d all 
Because when I use/ When setting up. sh to - d small, accessing fixed_index will list out of range
Looking forward to your reply, thank you
